src/utils/helpers.ts

```diff
@@ -29,31 +28,18 @@ export function checkNodeLinkable(
 
 export interface CoordProps {
   zoom: number;
-  scrollXY: [number, number];
-  layout: ElkRoot;
-  containerWidth: number;
-  containerHeight: number;
+  layoutXY: [number, number];
   containerRef: RefObject<HTMLDivElement | null>;
 }
 
 /**
  * Given various dimensions and positions, create a matrix
  * used for determining position.
  */
-export function getCoords({
-  zoom,
-  scrollXY,
-  layout,
-  containerWidth,
-  containerHeight,
-  containerRef
-}: CoordProps) {
+export function getCoords({ zoom, layoutXY, containerRef }: CoordProps) {
   const { top, left } = containerRef.current.getBoundingClientRect();
-  const offsetX = scrollXY[0] - containerRef.current.scrollLeft;
-  const offsetY = scrollXY[1] - containerRef.current.scrollTop;
-
-  const tx = (containerWidth - layout.width * zoom) / 2 + offsetX + left;
-  const ty = (containerHeight - layout.height * zoom) / 2 + offsetY + top;
+  const tx = layoutXY[0] - containerRef.current.scrollLeft + left;
```

Comment:
I'm wondering if we could add some unit tests here easily to make sure we don't break this again.

Reply:
Would you be willing to merge this without unit tests? I've moved on to a different project and no longer work with reaflow, so I'd like to get this merged and released so that my previous team can remove the patch.